lib/controller/page/profile_page_controller.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+import 'package:idea_tracker/view/dialog/confirm_logout_dialog.dart';
+import 'package:idea_tracker/view/page/splash_page.dart';
+
+class ProfilePageController extends ChangeNotifier {
+  int _tabs = 0;
+
+  onTabSelected() {}
+  _changePassword() {
+    return print("change password");
+  }
+
+  _logout() {
```

Comment:
logout should be:

public
async
It should call locator<AuthenticationService>().signOut()
It should alert a callback when the logout is complete